acquia_cms.profile

```diff
@@ -131,20 +123,7 @@ function acquia_cms_install_tasks(): array {
  */
 function install_acms_send_heartbeat_event() {
   $telemetry = Drupal::classResolver(AcquiaTelemetry::class);
-  $telemetry_service = \Drupal::service('acquia_connector.telemetry');
-  $config = \Drupal::config('cohesion.settings');
-  $cohesion_configured = $config->get('api_key') && $config->get('organization_key');
-  \Drupal::configFactory()
-    ->getEditable('acquia_connector.settings')
-    ->set('spi.amplitude_api_key', 'e896d8a97a24013cee91e37a35bf7b0b')
-    ->save();
-  $telemetry_service->sendTelemetry('acquia_cms_installed', [
-    'Application UUID' => Environment::getAhApplicationUuid(),
-    'Site Environment' => Environment::getAhEnv(),
-    'Install Time' => $telemetry->calculateTime('install_start_time', 'install_end_time'),
-    'Rebuild Time' => $telemetry->calculateTime('rebuild_start_time', 'rebuild_end_time'),
-    'Site Studio Install Status' => $cohesion_configured ? 1 : 0,
-  ]);
+  \Drupal::state()->set('acquia_cms.install_time', $telemetry->calculateTime('install_start_time', 'install_end_time'));
```

Comment:
@rajeshreeputra so this mean we will not be sending data for 1.5.x ?

Reply:
We are sending telemetry data using the [acquia_cms_common/src/EventSubscriber/KernelTerminate/AcquiaCmsTelemetry.php](https://github.com/acquia/acquia_cms/blob/develop/modules/acquia_cms_common/src/EventSubscriber/KernelTerminate/AcquiaCmsTelemetry.php) event subscriber of Acquia CMS Common module.